java-spiffe-core/src/main/java/spiffe/bundle/jwtbundle/JwtBundleSet.java

```diff
@@ -0,0 +1,73 @@
+package spiffe.bundle.jwtbundle;
+
+import lombok.NonNull;
+import lombok.Value;
+import lombok.val;
+import spiffe.bundle.BundleSource;
+import spiffe.exception.BundleNotFoundException;
+import spiffe.spiffeid.TrustDomain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * A <code>JwtBundleSet</code> represents a set of JWT bundles keyed by trust domain.
+ */
+@Value
+public class JwtBundleSet implements BundleSource<JwtBundle> {
+
+    ConcurrentHashMap<TrustDomain, JwtBundle> bundles;
```

Comment:
This should be `Map<TrustDomain, JwtBundle>` IMO unless there is some specific API of `ConcurrentHashMap<>` that we are using in this class.

Reply:
You are right. Fixed.